modules/tests/photons_control_tests/device_finder/test_filter.py

```diff
@@ -261,7 +261,7 @@ def assertFltrMatches(self, filtr, expect):
     describe "points":
         it "returns the InfoPoint enums for the fields that have values":
             filtr = Filter.from_kwargs(label="kitchen", product_id=22)
-            assert set(filtr.points) == set([InfoPoints.LIGHT_STATE, InfoPoints.VERSION])
+            assert set(filtr.points) == set([InfoPoints.LABEL, InfoPoints.VERSION])
```

Comment:
shouldn't a 22 get LIGHT_STATE, not LABEL?

Reply:
No, because it doesn't need to know if it's a light or not. It's filtering on label and product ID and that doesn't need a GetColor/is_light.

Reply:
right. nice.